pubsub/pstest/fake.go

```diff
@@ -114,7 +114,12 @@ type GServer struct {
 
 // NewServer creates a new fake server running in the current process.
 func NewServer(opts ...ServerReactorOption) *Server {
-	srv, err := testutil.NewServer()
+	return NewServerWithPort(0, opts...)
+}
+
+// NewServerWithPort creates a new fake server running in the current process at the specified port.
+func NewServerWithPort(port int, opts ...ServerReactorOption) *Server {
+	srv, err := testutil.NewServerWithPort(port)
 	if err != nil {
 		panic(fmt.Sprintf("pstest.NewServer: %v", err))
```

Comment:
nit: change this to NewServerWithPort. Perhaps add the port number if that's not logged in the error, I don't know if it is or not.

Reply:
`testutil.NewServerWithPort` just calls a basic `net.Listen` which should return an error containing the port